django/db/models/deletion.py

```diff
@@ -284,8 +284,8 @@ def delete(self):
 
             # update fields
             for model, instances_for_fieldvalues in self.field_updates.items():
-                query = sql.UpdateQuery(model)
                 for (field, value), instances in instances_for_fieldvalues.items():
+                    query = sql.UpdateQuery(model)
```

Comment:
That seems to be a `git am` leftover from [#29016](https://code.djangoproject.com/ticket/29016).